src/client/components/EventsPage/EventsPage.component.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { StudyGroupLogo } from '../StudyGroupLogo/StudyGroupLogo';
+import './EventsPage.styles.css';
+
+function EventsPage() {
+  return (
+    <div>
+      <div>Study Groups for Class15</div>
+      <div className="main-container">
```

Comment:
You should add a class to the root container named `events-page`. This will ensure that if someone else also names their component with the same name as you do you won't get problems.